source/frontend/src/features/documents/DocumentPreviewContainer.tsx

```diff
@@ -0,0 +1,167 @@
+import moment from 'moment';
+import { FC, useCallback, useContext, useEffect, useState } from 'react';
+
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { ModalContext } from '@/contexts/modalContext';
+import { ApiGen_Mayan_FilePage } from '@/models/api/generated/ApiGen_Mayan_FilePage';
+import { ApiGen_System_HttpStatusCode } from '@/models/api/generated/ApiGen_System_HttpStatusCode';
+
+import { DocumentViewerContext } from './context/DocumentViewerContext';
+import { IDocumentPreviewViewProps } from './DocumentPreviewView';
+import { createFileDownload } from './DownloadDocumentButton';
+import { useDocumentProvider } from './hooks/useDocumentProvider';
+
+export interface LoadedPages extends ApiGen_Mayan_FilePage {
+  loadedDocumentImageDataUrl?: string;
```

Comment:
q: is this the image Blob?

Reply:
No, this is the data url that was made from the blob.